addons/account_edi_ubl_cii/models/account_journal.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models, fields
+from odoo.addons.account_edi_ubl_cii.models.account_edi_common import COUNTRY_EAS
+
+
+class AccountJournal(models.Model):
+    _inherit = 'account.journal'
+
+    enable_ubl_cii_xml = fields.Boolean(string='BIS Billing', default=False)
+    can_enable_ubl_cii_xml = fields.Boolean(compute='_compute_can_enable_ubl_cii_xml')
+
+    @api.depends('type', 'company_id')
+    def _compute_can_enable_ubl_cii_xml(self):
+        for journal in self:
+            journal.can_enable_ubl_cii_xml = journal.type == 'sale' and\
+                journal.company_id.country_id.code in COUNTRY_EAS
```

Comment:
If we have one checkbox per format, I guess we can go through `_get_ubl_cii_builder` for each format to know whether it can be enabled or not.

Reply:
indeed!